style-guides/python.md

```diff
@@ -0,0 +1,32 @@
+# Python Infrastructure and Coding Standards 
```

Comment:
Let's follow a guidelines for docs too, like max-line-length for md files?
We can get to that after python I guess.

Reply:
Alight @Monal5031 good idea